config/chime_psr_verification.yaml

```diff
@@ -18,7 +18,7 @@ log_level: info
 num_elements: 2048
 num_local_freq: 1
 num_data_sets: 1
-samples_per_data_set: 49152
+samples_per_data_set: 3840
```

Comment:
Why was this switched?  Just to improve performance?

Reply:
That was just so I could run it quickly when checking the accuracy of answers.